phpunit.xml.dist

```diff
@@ -32,6 +32,7 @@ It's auto-generated by sonata-project/dev-kit package.
 
     <php>
         <ini name="precision" value="8" />
-        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=8" />
+        <server name="KERNEL_CLASS" value="Sonata\PageBundle\Tests\App\AppKernel" />
```

Comment:
well I was using this  `KERNEL_CLASS` to run the test from the IDE

Reply:
do you have some example for this `or use group legacy`?

Reply:
well regarding this I'll open a new Pull request, appeared many deprecations :'(

Reply:
`@group legacy` over some tests

Reply:
Nice, the deprecation problem was solved :) 

what is missing is this `KERNEL_CLASS` thing. I need this to run one of my tests

https://github.com/sonata-project/SonataPageBundle/runs/7131719464?check_suite_focus=true#step:8:33

Reply:
well I guess I saw some tests using command line, I will check how was it build and refactor my tests ;)

Reply:
@jordisala1991 Introduce multiple times tests with AppKernel.
the last one was https://github.com/sonata-project/SonataFormatterBundle/pull/672 if it can help you.

Reply:
@eerison Seems like you can do it directly in the classes: https://github.com/sonata-project/SonataMediaBundle/blob/f121156da9ea2e08e1ed872c70e278bac989925a/tests/Functional/Admin/GalleryAdminTest.php#L90-L96

Reply:
> @eerison Seems like you can do it directly in the classes: https://github.com/sonata-project/SonataMediaBundle/blob/f121156da9ea2e08e1ed872c70e278bac989925a/tests/Functional/Admin/GalleryAdminTest.php#L90-L96

ohhh, thanks for the info :)

Reply:
Nice @VincentLanglet  it works :)